### PR TITLE
Avoid global variables

### PR DESCRIPTION
**Pull request by Renegatto**
Just put functions into a class, and all main stuff into constructor.
Now you can create multiple instances of your bot at once.

If this doesn't work, give the feedback.

### CODE REVIEW

**Comment by qu-od** [comment]
Checking diffs

File: main.py [hunk 1]
```diff
@@ -24,255 +24,207 @@
...
+      reload(self.wheel)
+
+
+  def reload_commands(self):
+      bot.reload_extension('book_cmds')
+      bot.reload_extension('profile_cmds')
+      bot.reload_extension('match_cmds')
+
+
+  def load_bot_command_extensions(self):
```
Comment (qu-od) [inline main.py:225]: This function must be called on `__init__()`

Comment (qu-od) [reply main.py:225]: Oh, okay, I see it now. You've included it right before the `bot.run()` in the end of `__init__()`

File: main.py [hunk 2]
```diff
@@ -24,255 +24,207 @@
...
-
-# ------------------------ TEST COMMANDS ---------------------------------------
-@bot.group()
-async def test(ctx: commands.Context):
-    """
-    checks if a subcommand is being invoked.
-    """
-    if ctx.invoked_subcommand is None:
-        await ctx.send(f"Command {ctx.subcommand_passed} haven't been invoked")
+class Bot:
```
Comment (qu-od) [inline main.py:32]: Don't you want to inherit our Bot from the Bot baseclass directly? Do you mean that composition is better than inheritance?

File: main.py [hunk 3]
```diff
@@ -24,255 +24,207 @@
...
-        + " VALUES ('{name}', {uid}, {sid}, {is_private}, '{privileges}');".format(
-            name=ctx.author.name,
-            uid=ctx.author.id,
-            sid=(ctx.guild.id if ctx.guild else "NULL"),
-            is_private=False,
-            privileges=[]
-        )
+    self.bot = commands.Bot(
+        command_prefix=commands.when_mentioned_or("!m "),
+        description=description,
```
Comment (qu-od) [inline main.py:46]: I thought `description` and `debug_guilds` are also counted as global variables. Aren't they?

File: main.py [hunk 4]
```diff
@@ -24,255 +24,207 @@
...
-    bot.load_extension('match_cmds')
-
-
-load_bot_command_extensions()
-bot.run(os.getenv("MAEVE_TOKEN"))
-
-
+    self.test_books = BookList()
+
+    bot.event(self.on_ready)
```
Comment (qu-od) [inline main.py:52]: De-sugared version of command-registering functions is complicated. 

- I
- want
- those
- decorators
- back.

**Review by qu-od — APPROVED** [review]
Anyway. I will merge every line into staging, and also will test it now. I hope it works.

File: main.py (hunk 3 above)
Comment (Renegatto) [reply main.py:46]: They're constants, constants can be global.

File: main.py (hunk 2 above)
Comment (Renegatto) [reply main.py:32]: No, I don't mean. This class is unrelated to original Hierarchy.

File: main.py (hunk 4 above)
Comment (Renegatto) [reply main.py:52]: Use the decorator function not as decorator, boi

**Pull request closed without merging**